sim/midas/src/main/cc/bridges/bridge_driver.h

```diff
@@ -5,6 +5,18 @@
 
 #include "simif.h"
 
+namespace BridgeConstants {
+/**
+ *  @brief Logical byte width of Bridge streams
+ *
+ * Bridge streams are logically latency-insensitive FIFOs with a width of
+ * \c STREAM_WIDTH_BYTES.
+ * \note { The host-implementation may use a different width under-the-hood but
+ * this should not be exposed to bridge developers. }
+ */
```

Comment:
Non-blocking question: Are these comments rendered somewhere? What is the format for these types of comments?

Reply:
These are doxygen. With the C++ Extension pack, `VSCode` will provide a template and then render them as a tooltip, and elsewhere in the tooling. We could also one day build docs as we do for Scala and host them somewhere.